src/Shell.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Laravel\Tinker;
+
+use ErrorException;
+use Exception;
+use Illuminate\Support\Str;
+use Psy\CodeCleaner\NoReturnValue;
+use Psy\Exception\BreakException;
+use Psy\Exception\Exception as PsyException;
+use Psy\Shell as BaseShell;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class Shell extends BaseShell
+{
+    const BUFF_PROMPT = '  ';
+    const PROMPT = '> ';
+
+    /**
+     * The OutputStyle instance.
+     *
+     * @var \Symfony\Component\Console\Style\SymfonyStyle
+     */
+    protected $output;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function doRun(InputInterface $input, OutputInterface $output): int
+    {
+        return parent::doRun($input, $this->output = new SymfonyStyle($input, $output));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getHeader(): string
+    {
+        return '';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function formatException(Exception $exception): string
+    {
+        $message = $exception->getMessage();
+
+        if (Str::contains($message, 'psy/psysh/src/ExecutionLoopClosure.php')) {
```

Comment:
This one [should already be handled in PsySH](https://github.com/bobthecow/psysh/blob/main/src/Shell.php#L1214). Let's fix the bug there instead?